### ISSUE TITLE
New translation: Georgian

### ISSUE THREAD

**Issue by demonno** [squidfunk/mkdocs-material#1963]
## Instructions

1. Check, if your language is already available: [here](https://bit.ly/3c2ox0C)
2. If it isn't, please translate the labels on the right:

``` jinja
{% macro t(key) %}{{ {
  "language": "ka",
  "direction": "ltr",
  "clipboard.copy": "კოპირება",
  "clipboard.copied": "კოპირებულია",
  "edit.link.title": "გვერდის რედარქირება",
  "footer.previous": "წინა",
  "footer.next": "შემდეგი",
  "meta.comments": "კომენტარები",
  "meta.source": "წყარო",
  "nav.title": "ნავიგაცია",
  "search.config.lang": "ka",
  "search.config.pipeline": "trimmer, stopWordFilter",
  "search.config.separator": "[\s\-]+",
  "search.placeholder": "ძებნა",
  "search.reset": "გასუფთავება",
  "search.result.placeholder": "ჩაწერე ძებნის დასაწყებად",
  "search.result.none": "დოკუმენტი ვერ მოიძებნა",
  "search.result.one": "მოიძებნა 1 დოკუმენტი",
  "search.result.other": "მოიძებნა # დოკუმენტი",
  "search.result.more.one": "კიდევ 1 ამ გვერდზე",
  "search.result.more.other": "კიდევ # ამ გვერდზე",
  "skip.link.title": "კონტენტზე გადასვლა",
  "source.link.title": "საცავში გადასვლა",
  "source.revision.date": "ბოლო განახლება",
  "tabs.title": "ტაბები",
  "toc.title": "სარჩევი"
}[key] }}{% endmacro %}
```

Thanks!

**Comment by squidfunk** [comment]
Added in 62484128 - thanks for your efforts!

**Comment by squidfunk** [comment]
Released as part of 6.0.2
